docs/docs/tailwind-css.md

```diff
@@ -13,7 +13,7 @@ There are two ways you can use Tailwind with Gatsby:
 
 You have to install and configure Tailwind for both of these methods, so this guide will walk through that step first, then you can follow the instructions for either PostCSS or CSS-in-JS.
 
-## Installing and Configuring Tailwind
+## Installing and configuring Tailwind
```

Comment:
Sentence case applies to these headings, as an FYI. Title case is for page titles: https://www.gatsbyjs.org/contributing/gatsby-style-guide/#format-titles-and-headers

Reply:
@marcysutton not sure I follow - isn't this sentence case?

Reply:
Yeah, you're correct @samrae7